packages/react/src/components/Stack/definitions.ts

```diff
@@ -0,0 +1,11 @@
+import { spacing } from '@carbon/layout';
+
+/**
+ * The steps in the spacing scale
+ * @type {Array<number>}
+ */
+export const SPACING_STEPS = Array.from({ length: spacing.length - 1 }).map(
+  (_, step) => {
+    return step + 1;
+  }
+);
```

Comment:
To my knowledge this does not (and likely will never) be used outside of the context of `Stack`. Is there a reason requiring this to be in a separate file?